clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -416,6 +438,14 @@ public void send(String method, Object... args) {
         sendHubMessage(invocationMessage);
     }
 
+    /**
+     * Invokes a hub method on the server using the specified name and arguments.
+     * @param returnType The expected return type.
+     * @param method The name of the server method to invoke.
+     * @param args The arguments used to invoke the server method.
+     * @param <T> The expected return type.
+     * @return A Single of the expected return type.
```

Comment:
Suggest: "A Single that yields the return value when the invocation has completed"